ovn-tester/cms/__init__.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2023 Canonical
```

Comment:
I'm not sure about the right way to do this.  Aren't we mostly reusing code that already existed?  Until now the only place where we mentioned the copyright and licenses was in the `copyright` files.  Do we still need to add it to all files?  If so, should we also add all other copyright owners, i.e.:

https://github.com/ovn-org/ovn-heater/blob/648b221349f5bd2d8f51191d7443ad216c54a51c/copyright#L20-L24C1

Reply:
Ok, but then you should probably add Canonical to `copyright`.  You are also contributing new code.